docs/index.md

```diff
@@ -16,6 +16,7 @@ open_source/installation_library/index
 open_source/scan/index
 open_source/customize_tests/index
 open_source/integrate_tests/index
+open_source/testset_generation/index
```

Comment:
Could you put it right after scan given its importance?

Reply:
Sure!